### PR TITLE
Feet/api stubs

### LINKED ISSUE #19: Feature Request: Back-end file structure
**User Story**
For example: 
As a back end developer
I want the server to have an appropriate file structure

**Priority**
[]: High
[x]: Medium
[]: Low

**Describe the solution you want to implement**
Endpoint API is split over multiple classes and files

**Describe alternatives you've considered**
Doing the opposite and having all API logic in the server.py class

**Issue Dependencies**
none

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by tant836**
---
name: server api stubs
about: stubs for server
---

Closes #(19)
**Summary**
This PR introduces file structure and stubs for the server

New files are for endpoints of the server

**Testing**
Test all the end points work, they should return 1 - 4 

**Checklist**

[x] - Code builds successfully
[] - All tests pass

### CODE REVIEW

**Comment by lken274** [comment]
requested review from whole back-end team because this is quite fundamental to continuing work

**Comment by tant836** [comment]
code builds

File: server/src/api/bill_data.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import cherrypy
+import urllib.request
+import json
+import base64
```
Comment (lken274) [inline server/src/api/bill_data.py:4]: nit: unused imports

File: server/src/main.py [hunk 2]
```diff
@@ -31,8 +32,10 @@ def run_main_app():
         'base_path': os.getcwd()
     }
     
-    cherrypy.tree.mount(server.ApiServer(), "/api/", conf)
-    cherrypy.tree.mount(server.MainApp(), "/", conf)
+    #cherrypy.tree.mount(server.ApiServer(), "/api/", conf)
```
Comment (lken274) [inline server/src/main.py:35]: nit: remove commented out code

**Review by lken274 — APPROVED** [review]
LGTM. Endpoint stubs seem to match api description.

**Review by liappi — APPROVED** [review]
LGTM

**Pull request merged**